cdap-ui/app/directives/dag-plus/my-dag.less

```diff
@@ -136,6 +137,109 @@ my-dag-plus {
         }
       }
 
+      &.action {
+        @brown: #988470;
```

Comment:
May be at the top of the file? Took a little while to see that this is a declaration.